src/main/java/com/linkedin/kmf/services/ConsumeService.java

```diff
@@ -45,6 +39,14 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
+import com.linkedin.kmf.common.DefaultTopicSchema;
```

Comment:
Configure your IDE such that you don't have a change in import order. This will help keep the diff minimal and relevant.